src/blocks/card.css

```diff
@@ -0,0 +1,109 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", serif;
+}
+
+.card__product-img {
+  display: block;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  display: block;
```

Comment:
```suggestion
```
redundant. it is default value for h2
![Screenshot from 2023-04-30 18-29-37](https://user-images.githubusercontent.com/33029864/235361734-2c89cbd3-5198-4453-bbfb-73369686d51c.png)